data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
+      [ "Уиттен", "Whitten" ],
+      [ "Уиттингтон", "Whittington" ],
+      [ "Смарт", "Smart" ],
+      [ "Уоркмен", "Workman" ],
+      [ "Уотерман", "Waterman" ],
+      [ "Уэбстер", "Webster" ],
+      [ "Уэллетт", "Ouellette" ],
+      [ "Уэлч", "Welch" ],
+      [ "Фалкон", "Falcon" ],
+      [ "Фаррис", "Farris" ],
+      [ "Фейр", "Fair" ],
+      [ "Фитцпатрик", "Fitzpatrick" ],
+      [ "Флеминг", "Fleming" ],
+      [ "Фой", "Foy" ],
+      [ "Фоли", "Foley" ],
+      [ "Фоллон", "Fallon" ],
+      [ "Фомэн", "Forman" ],
+      [ "Фонтено", "Fontenot" ],
+      [ "Траут", "Trout" ],
+      [ "Франк", "Frank" ],
+      [ "Франкс", "Franks" ],
+      [ "Франс", "France" ],
+      [ "Фрейли", "Fraley" ],
+      [ "Фрейтас", "Freitas" ],
+      [ "Фримен", "Freeman" ],
+      [ "Фуртадо", "Furtado" ],
+      [ "Фэган", "Fagan" ],
+      [ "Хадсон", "Hudson" ],
+      [ "Хайтауер", "Hightower" ],
+      [ "Хакетт", "Hackett" ],
+      [ "Хамель", "Hamel" ],
+      [ "Хаммондс", "Hammonds" ],
+      [ "Харгроув", "Hargrove" ],
+      [ "Хатсон", "Hutson" ],
+      [ "Хау", "Howe" ],
+      [ "Хаятт", "Hyatt" ],
+      [ "Хейден", "Hayden" ],
+      [ "Хейн", "Hein" ],
+      [ "Хейр", "Hare" ],
+      [ "Хеллер", "Heller" ],
+      [ "Хелмс", "Helms" ],
+      [ "Хелтон", "Helton" ],
+      [ "Хенди", "Handy" ],
+      [ "Херли", "Hurley" ],
+      [ "Херрик", "Herrick" ],
+      [ "Херрон", "Herron" ],
+      [ "Хиггинботэм", "Higginbotham" ],
+      [ "Хинкль", "Hinkle" ],
+      [ "Хинонес", "Quinones" ],
+      [ "Коттон", "Cotton" ],
+      [ "Холл", "Hall" ],
+      [ "Хоули", "Hawley" ],
+      [ "Кристиан", "Christian" ],
+      [ "Хэлм", "Helm" ],
+      [ "Хэнкинс", "Hankins" ],
+      [ "Хёрн", "Hearn" ],
+      [ "Флауэрс", "Flowers" ],
+      [ "Чёрч", "Church" ],
+      [ "Чайлдс", "Childs" ],
+      [ "Чайлдресс", "Childress" ],
+      [ "Шампейн", "Champagne" ],
+      [ "Чампион", "Champion" ],
+      [ "Чаппелл", "Chappell" ],
+      [ "Чемберс", "Chambers" ],
+      [ "Чен", "Chen" ],
+      [ "Ченг", "Cheng" ],
+      [ "Чендлер", "Chandler" ],
+      [ "Чепмен", "Chapman" ],
+      [ "Хек", "Heck" ],
+      [ "Скейлс", "Scales" ],
+      [ "Читэм", "Cheatham" ],
+      [ "Пейс", "Pace" ],
+      [ "Шанс", "Chance" ],
+      [ "Шарп", "Sharp" ],
+      [ "Майнер", "Miner" ],
+      [ "Швитцер", "Switzer" ],
+      [ "Шейвер", "Shaver" ],
+      [ "Шекелфорд", "Shackelford" ],
+      [ "Шенк", "Shank" ],
+      [ "Шепхерд", "Shepherd" ],
+      [ "Шерил", "Sherrill" ],
+      [ "Шиа", "Shea" ],
+      [ "Шиитс", "Sheets" ],
+      [ "Шихан", "Sheehan" ],
+      [ "Шоемейкер", "Shoemaker" ],
+      [ "Коттер", "Cotter" ],
+      [ "Штейн", "Stein" ],
+      [ "Штраус", "Strauss" ],
+      [ "Эддисон", "Addison" ],
+      [ "Эйкен", "Aiken" ],
+      [ "Эллиотт", "Elliott" ],
+      [ "Эскамилья", "Escamilla" ],
+      [ "Эскивель", "Esquivel" ],
+      [ "Эспарса", "Esparza" ],
+      [ "Эхолс", "Echols" ],
```

Comment:
Should be `Эколс`, like [`Лейтан Эколс`](https://ru.wikipedia.org/wiki/Lil_Mosey), better known as Lil Mosey.

Worth noting that Russian Wikipedia also lists this surname as [`Эчолс`](https://ru.wikipedia.org/wiki/%D0%AD%D1%87%D0%BE%D0%BB%D1%81), but I believe this to be an incorrect transcription, given how the word *echo* is pronounced. Then again, it could be one of those names that are pronounced differently, even with the same spelling.